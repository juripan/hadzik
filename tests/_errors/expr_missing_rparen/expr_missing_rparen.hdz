naj missing_rparen = (8
