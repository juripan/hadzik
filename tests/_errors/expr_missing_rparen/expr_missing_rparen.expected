Failed here: tests/_errors/expr_missing_rparen/expr_missing_rparen.hdz:1:24
naj missing_rparen = (8
                       ^
[31mSyntaxError[0m: (line 1 column 24) expected ')'
